src/main/java/org/commcare/suite/model/Detail.java

```diff
@@ -60,6 +60,7 @@ public class Detail implements Externalizable {
     private TreeReference nodeset;
 
     private DisplayUnit title;
+    private Text noItemsText;
```

Comment:
it would be good to annotate vat declaration here and `getNoItemsText` below as `@Nullable` so that we get a warning if we don't do a `null` check on this.

Reply:
Added in commit [07bf910](https://github.com/dimagi/commcare-core/pull/1209/commits/07bf9103af7a0c0ed386b4991d3d7757371d345b)